lib/invoiced/payment_source.rb

```diff
@@ -0,0 +1,27 @@
+module Invoiced
+    class PaymentSource < Object
+        include Invoiced::Operations::List
+        include Invoiced::Operations::Create
+        include Invoiced::Operations::Delete
+
+        OBJECT_NAME = 'payment_source'
+
+        def delete
+            if object == 'card'
+                @endpoint = '/cards/' + id.to_s
+            end
+            if object == 'bank_account'
+                @endpoint = '/bank_accounts/' + id.to_s
+            end
+
+            if super
+                return true
+            else
+                @endpoint = '/payment_sources' + id.to_s
```

Comment:
I think there's supposed to be a forward slash here?

Reply:
Or maybe no `id.to_s`?

Reply:
yep

Reply:
one moment

Reply:
I'm trying to understand the purpose of this line. What does it do?

Reply:
No longer relevant after further changes